pkg/bmc/ilo4.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2016-2018 Hewlett Packard Enterprise Development LP
+
+package bmc
+
+import (
+	"net/url"
+)
+
+func init() {
+	registerFactory("ilo4", newILOAccessDetails, []string{})
```

Comment:
I think you should still pass `[]string{"https"}` for the schemes, so that `ilo4+https://` works as well as `ilo4://`.

Reply:
@zaneb  Thanks for the comment. Made the changes.